### PR TITLE
Fix: Sanitise Form 429 Error

### PR DESCRIPTION
**Pull request by christopher-semmens**

### CODE REVIEW

File: packages/forms-web-app/src/lib/decode-string.js [hunk 1]
```diff
@@ -1,7 +1,13 @@
 const decodeString = (stringToDecode) => {
 	if (!stringToDecode || typeof stringToDecode !== 'string') return '';
 
-	const decodedString = decodeURIComponent(stringToDecode);
+	let decodedString = '';
+
+	try {
+		decodedString = decodeURIComponent(stringToDecode);
+	} catch {
+		decodedString = stringToDecode;
```
Comment (mauriciokin) [inline packages/forms-web-app/src/lib/decode-string.js:9]: Would be the optimal expected behaviour to return the primary payload if it couldn't be decoded (catch error)?

File: packages/forms-web-app/src/lib/encode-string.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 const encodeString = (stringToEncode) => {
 	if (!stringToEncode || typeof stringToEncode !== 'string') return '';
 
-	const encodedString = encodeURIComponent(stringToEncode);
+	let encodedString = '';
+
+	try {
+		encodedString = encodeURIComponent(stringToEncode);
+	} catch {
+		encodedString = stringToEncode;
```
Comment (mauriciokin) [inline packages/forms-web-app/src/lib/encode-string.js:9]: Would be the optimal expected behaviour to return the primary payload if it couldn't be decoded (catch error)? Doesn't it have a compounded effect when it goes through the rest of the steps?

File: packages/forms-web-app/src/lib/decode-string.js (hunk 1 above)
Comment (christopher-semmens) [reply packages/forms-web-app/src/lib/decode-string.js:9]: This fixes an error when an invalid input was being decoded (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Malformed_URI). In user cases we still want the string but for the decode/encode to not take place. Looking into creating checks for valid strings looks quite cumbersome. This was the most elegant solution that I could find.

**Review by mauriciokin — APPROVED** [review]

**Review by mauriciokin — APPROVED** [review]
Legit

**Pull request merged**